charts/sptribs-case-api/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for sptribs-case-api App
 name: sptribs-case-api
 home: https://github.com/hmcts/sptribs-case-api
-version: 0.0.86
+version: 0.0.87
```

Comment:
this need to be changed if case linking is released before this.

Reply:
1st we plan to deploy case flags, so to keep it.